.github/workflows/release.yml

```diff
@@ -3,24 +3,34 @@ name: Main
 on:
   push:
     tags:
-      - 'v*.*.*'
+      - "v*.*.*"
 
 jobs:
   build:
     strategy:
       matrix:
         rust-toolchain: [stable]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        arch: [amd64]
+        os: [ubuntu-latest, macos-11, windows-latest]
```

Comment:
Just to clarify for anyone unfamiliar (such as myself), where the PR description states:

> Note that macos-11 is not the same as macos-latest, and is required to get a recent Xcode version.

It means `macos-11` is actually more recent than `macos-latest`, as documented officially here:

https://github.com/actions/virtual-environments#available-environments